api.planx.uk/server.test.js

```diff
@@ -30,6 +30,17 @@ it("mocks hasura", async () => {
     });
 });
 
+describe("authentication", () => {
+  test("Failed login endpoint", async () => {
+    await supertest(app)
+      .get("/auth/login/failed")
+      .expect(401)
+      .then((res) => {
+        expect(res.body).toEqual({ error: "user failed to authenticate." });
+      });
+  })
+});
```

Comment:
Apologies for going for low hanging fruit here - I'll pick a module or two later this week and bump up coverage 👍